### ISSUE TITLE
SMTP Email Error

### ISSUE THREAD

**Issue by fabiofolino** [tendenci/tendenci#1298]
I'm having issues getting SMTP email working. I've been testing using the password reset, but this happens even when attempting to send a test email in the User section.

These are my settings below

```
DEFAULT_FROM_EMAIL = 'no-reply@XXXXX.com'
SERVER_EMAIL = DEFAULT_FROM_EMAIL

EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
EMAIL_USE_TLS = True
EMAIL_HOST = 'mail.XXXXX.com'
EMAIL_PORT = 465
EMAIL_HOST_USER = 'no-reply@XXXXX.com'
EMAIL_HOST_PASSWORD = 'XXXXX'
```
This is what I'm seeing in the error log. 
```
TIME="2024-07-20 11:09:19,118" LEVEL=WARNING PID=1457403 LOGGER="py.warnings" MSG="/usr/lib/python3.10/socket.py:776: ResourceWarning: unclosed <socket.socket fd=17, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM,
proto=6, laddr=('XXX.XXX.XXX.XXX', 36552), raddr=('XXX.XXX.XXX.XXX', 465)>
  self._sock = None
"
TIME="2024-07-20 11:09:20,034" LEVEL=WARNING PID=1457712 LOGGER="py.warnings" MSG="/srv/venv_mysite/lib/python3.10/site-packages/haystack/__init__.py:4: DeprecationWarning: pkg_resources is deprecated as an API.
See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import DistributionNotFound, get_distribution, parse_version
"
TIME="2024-07-20 11:09:20,048" LEVEL=WARNING PID=1457712 LOGGER="py.warnings" MSG="/srv/venv_mysite/lib/python3.10/site-packages/pkg_resources/__init__.py:3117: DeprecationWarning: Deprecated call to
`pkg_resources.declare_namespace('zope')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
"
TIME="2024-07-20 11:09:21,871" LEVEL=WARNING PID=1457712 LOGGER="py.warnings" MSG="/srv/venv_mysite/lib/python3.10/site-packages/haystack/fields.py:5: RemovedInDjango50Warning: The django.utils.datetime_safe module is deprecated.
  from django.utils import datetime_safe
"

```

**Comment by jennyq** [comment]
Hi @fabiofolino, 

Those are just warnings, not errors. Maybe you can start a django shell "Python manage.py shell", try the code like below and see what errors you get:

    from django.core.mail import send_mail

    send_mail(
        "Subject here",
        "Here is the message.",
        "from@example.com",
        ["to@example.com"],
        fail_silently=False,
    )

**Comment by fabiofolino** [comment]
Thanks. That was helpful. It seems the settings I had were incorrect. I needed to change to EMAIL_USE_SSL instead of EMAIL_USE_TLS.

The only issue now is that it doesn't seem to be pick up my DEFAULT_FROM_EMAIL setting. noreply@example.com is not what it is set to. I restarted the server to double check and it still gives this error below. Is there somewhere else to set this other than in setting.py?

```
SMTPRecipientsRefused at /accounts/password/reset/
{'xxxx@gmail.com': (550, b'Verification failed for <noreply@example.com>\nThe mail server does not recognize noreply@example.com as a valid sender.\nSender verify failed')}
```
